### PR TITLE
add debug_traceCall

### PR DESCRIPTION
**Pull request by yoomee1313**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/dapp/json-rpc/api-references/debug.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ The enabled APIs are as follows:
 - [debug_traceBlockByNumberRange](./debug/tracing.md#debug_traceblockbynumberrange)
 - [debug_traceBlockFromFile](./debug/tracing.md#debug_traceblockfromfile)
 - [debug_traceTransaction](./debug/tracing.md#debug_tracetransaction)
+- [debug_tracecall](./debug/tracing.md#debug_tracecall)
```
Comment (scott-klaytn) [inline docs/dapp/json-rpc/api-references/debug.md:73]: "debug_traceCall"? Not using camelcase here is intended?

File: docs/dapp/json-rpc/api-references/debug/tracing.md [hunk 2]
```diff
@@ -420,6 +420,36 @@ $ curl -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"de
...
 ```
 
+## debug_traceCall <a id="debug_tracecall"></a>
+The `traceCall` Returns the tracing result by executing a klay call within the context of the given block execution.
```
Comment (scott-klaytn) [inline docs/dapp/json-rpc/api-references/debug/tracing.md:424]: "The `traceCall` Returns" -> "The `traceCall` returns" or "Returns"

**Review by scott-klaytn — CHANGES_REQUESTED** [review]

File: docs/dapp/json-rpc/api-references/debug/tracing.md (hunk 2 above)
Comment (yoomee1313) [reply docs/dapp/json-rpc/api-references/debug/tracing.md]: Thanks! fixed at https://github.com/klaytn/klaytn-docs/pull/591/commits/3bd69fc0cef21cff0569207a404c54977bc747c7

File: docs/dapp/json-rpc/api-references/debug.md (hunk 1 above)
Comment (yoomee1313) [reply docs/dapp/json-rpc/api-references/debug.md]: Fixed at https://github.com/klaytn/klaytn-docs/pull/591/commits/3bd69fc0cef21cff0569207a404c54977bc747c7

**Review by scott-klaytn — APPROVED** [review]

**Review by ayo-klaytn — APPROVED** [review]

**Pull request merged**